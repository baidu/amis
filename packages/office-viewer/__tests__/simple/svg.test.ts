import {snapShotTest} from '../snapShotTest';

test('svg', async () => {
  snapShotTest('./docx/simple/svg.xml');
});
