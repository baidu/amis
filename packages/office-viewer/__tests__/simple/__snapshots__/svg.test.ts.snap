// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`svg 1`] = `
<article>
  <section
    style="position: relative; width: 815.98px; padding: 96.00px 96.00px 96.00px 96.00px;"
  >
    <p
      class="docx-viewer-0-p"
      style="position: relative;"
    >
      <span
        class="docx-viewer-0-r"
      >
        <div
          data-id="2"
          data-name="Graphic 2"
          style="display: inline-block; width: 623.98px; height: 623.98px;"
        >
          <img
            alt=""
            src="blob:http://localhost/mock"
            style="position: relative; left: 0.00px; top: 0.00px; width: 623.98px; height: 623.98px;"
          />
        </div>
      </span>
    </p>
  </section>
</article>
`;
